docs/user/customization.md

```diff
@@ -0,0 +1,122 @@
+
+## Customization Levels
+
+The various customization that the installer enables can be broken into four major levels: OpenShift, Kubernetes, Platform, and OS. These four levels are rough abstraction layers (OpenShift being the highest layer and OS being the lowest) and fall into either the managed or unmanaged buckets. The levels within the managed bucket (OpenShift and Platform) encompass customization that is safe to perform - installation and automatic updates will succeed regardless of the changes made (to a reasonable degree). The levels within the unmanaged bucket (Kubernetes and OS) encompass customization that is not necessarily safe - after introducing changes, installation and automatic updates may not succeed.
+
+### OpenShift Customization
+
+The most simple customization is exposed by the installer as an interactive series of prompts. These prompts are required and represent the lowest degree of customization. They are needed in order to get a running OpenShift cluster, but they aren't enough to get anything other than a vanilla deployment out of the box. Further customization is possible once the cluster has been provisioned, but isn't covered in this document as it is a "Day 2" operation.
+
+### Platform Customization
+
+While the default cluster size may be sufficient for some, many will need to make alterations. This can include increasing the number of machines in the control plane, changing the type of the virtual machines that will be used (e.g. AWS instances), or adjusting the CIDR range used for the Kubernetes service network. This level of customization is exposed via the installer's `install-config.yml`. The install-config can be accessed by running `openshift-install create install-config`. This file can then be modified as needed before running a later target.
+
+An example `install-config.yml` is shown below. This configuration has been modified to show the customization that is possible via the install config. By default, the generated `install-config.yml` will not have all of these fields populated, so they will need to be manually added if they are needed.
+
+```yaml
+apiVersion: v1beta1
+
+pullSecret: "auths": ...
+sshKey: ssh-ed25519 AAAA...
+
+baseDomain: example.com
+metadata:
+  name: test-cluster
+
+machines:
+- name: master
+  replicas: 3
+  platform:
+    zones:
+    - us-west-2a
+    - us-west-2b
+- name: worker
+  replicas: 5
+  platform:
+    type: c5.9xlarge
+    iamRoleName: elastictranscoder-access
+    rootVolume:
+      iops: 4000
+      size: 500
+      type: io1
+    zones:
+    - us-west-2c
+
+networking:
+  clusterNetworks:
+  - cidr: 10.128.0.0/14
+    hostSubnetLength: 9
+  machineCIDR: 10.0.0.0/16
+  serviceCIDR: 172.30.0.0/16
+  type: OpenshiftSDN
+
+platform:
+  aws:
+    region: us-west-2
+    userTags:
+      costCenter: 7536
+      adminContact: jdoe
+```
+
+#### AWS Platform Customization
```

Comment:
It would be good to put this under `docs/user/aws/` for better platform isolation and potential `OWNERS` scoping.